server/api/cartItems.js

```diff
@@ -1,36 +1,88 @@
-const express = require('express');
+const express = require("express");
 const app = express();
-const { CartItem } = require('../db/index');
+const { CartItem } = require("../db/index");
 
-app.get('/', (req, res, next) => {
+// these two routes are mostly for clarity but do add some functionality. Especially when it comes to decremeting the quantity of a cartItem.
+
+// increment
```

Comment:
Let's place all 'put' requests together, put :id , put :id/increment, put :id/decrement

Reply:
Do you mean in terms of where they're in the file?

Reply:
yes, just to help with readability and finding the routes where expected for troubleshooting purposes in the future.